pkg/kev/testdata/reconcile-healthcheck-http/docker-compose.kev.dev.yaml

```diff
@@ -0,0 +1,26 @@
+version: "3.7"
+services:
+  db:
+    labels:
+      kev.service.type: None
+      kev.workload.cpu: "0.1"
+      kev.workload.image-pull-policy: IfNotPresent
+      kev.workload.liveness-probe-type: http
+      kev.workload.liveness-probe.http-path: /status
```

Comment:
all liveness probe config keys should have hyphens - see my comment above regarding config keys.

Reply:
These will need to match whatever is defined above, I'll update everything after because are used in all examples and tests.